hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/fsdataset/impl/FsDatasetImpl.java

```diff
@@ -1068,9 +1070,8 @@ static File moveBlockFiles(Block b, ReplicaInfo replicaInfo, File destdir)
           + srcReplica + " metadata to "
           + dstMeta, e);
     }
-    if (LOG.isDebugEnabled()) {
-      LOG.info("Linked " + srcReplica.getBlockURI() + " to " + dstFile);
-    }
+    LOG.info("Linked {} to {} . Dest meta file: {}",
```

Comment:
Should be debug or trace level?

Reply:
This seems a bug, we are guarding with isDebugEnabled and if DEBUG is enabled, we are logging at INFO level. Either we can use INFO or DEBUG directly.